README.md

```diff
@@ -103,8 +103,11 @@ Once you have confirmed access, you can run the following command to download th
 ```bash
 # Get your HF token from https://huggingface.co/settings/tokens
 
-# Llama 3.1 tokenizer.model
-python scripts/download_tokenizer.py --repo_id meta-llama/Meta-Llama-3.1-8B --tokenizer_path "original" --hf_token=...
+# Llama 3.1 tokenizer (automatically downloads original/tokenizer.model)
+python scripts/download_tokenizer.py --repo_id meta-llama/Meta-Llama-3.1-8B --hf_token=...
+
+# DeepSeek tokenizer (automatically downloads tokenizer.json and tokenizer_config.json)
+python scripts/download_tokenizer.py --repo_id deepseek-ai/DeepSeek-V3
```

Comment:
Just curious, is `hf_token` not required?

Reply:
yup good Q, it is MIT license so no token required: https://huggingface.co/deepseek-ai/DeepSeek-V3/blob/main/tokenizer.json